lib/ledger_sync/adaptors/quickbooks_online/deposit/searcher.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require_relative '../searcher'
+
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Deposit
+        class Searcher < QuickBooksOnline::Searcher
+          def resources
+            @resources ||= begin
+              adaptor
+                .query(
+                  resource: 'deposit',
+                  limit: limit,
+                  offset: offset
+                )
+                .map do |c|
+                  LedgerSync::Deposit.new(
+                    ledger_id: c.fetch('Id'),
+                    name: c.dig('Id')
```

Comment:
Is this a typo? Or are you just saying the ID is the name?

Reply:
Actually in this case it doesnt make sense to set name. As thats not an attribute. Name is computed property.